pages/contact.html

```diff
@@ -33,14 +33,20 @@
   <link rel="stylesheet" href="../assets/css/styles.css">
   <link rel="stylesheet" href="../assets/css/contact.css">
 
+
+  <script
+  id="mcjs">!function (c, h, i, m, p) { m = c.createElement(h), p = c.getElementsByTagName(h)[0], m.async = 1, m.src = i, p.parentNode.insertBefore(m, p) }(document, "script", "https://chimpstatic.com/mcjs-connected/js/users/8773dffc3f528b0b18fd7aef8/3038d5447a8da88cd34cc38f5.js");</script>
+
 </head>
 
 <body>
   <!-- site header -->
   <header id="site-header" class="fixed-top">
     <nav class="navbar navbar-expand-lg navbar-dark">
-      <a class="navbar-brand" href="index.html">
-        <span class="fa fa-shield"></span> Captivate
+      <a class="navbar-brand" href="../../index.html">
+        <span>
+          <img src="../assets/images/logos/favicons/favicon.ico">
+        </span> UWB ACM
```

Comment:
I have made a new issue for redesigning the header #287 

You can leave these header changes for the time being, but please don't make any more changes.